todo_app/models/view_model.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
+import os
+
+
+class ViewModel:
+
+    def __init__(self, items):
+         self._items = items
+    
+    @property
+    def items(self):
+        return self._items
+    
+    
+    @property
+    def to_do_items(self):
+        to_doings = [this_item for this_item in self.items if this_item.status == os.getenv('DEFAULT_TO_DO_NAME')]
```

Comment:
I think this line here (and the equivalents for doing/done) is essentially why your tests are failing when run directly - running `poetry run pytest` _doesn't_ pick up any environment variables by default, so it's not matching the statues you're setting (e.g. `To Do`) since the check is against `None`

Adding the following lines to the top of your `test_app.py` file would be one way to solve this; actively loading in those test statuses for your unit tests:
```
from dotenv import load_dotenv, find_dotenv

file_path = find_dotenv('.env.test')
load_dotenv(file_path, override=True)
```

Reply:
https://github.com/ianm1968/ToDoTake2/commit/43b7f382f6629de6ca6dd21eba4b1cf48462b78c